src/pages/StudentWebcam.tsx

```diff
@@ -213,6 +336,7 @@ const StudentWebcam = () => {
       // participant.tracks.forEach((publication) => {
       //   if (publication.track.kind === "video" && localVideoRef.current) {
       //     publication.track.attach(localVideoRef.current);
```

Comment:
If this is deprecated, it should probably be removed if we have no use for it

Reply:
removed